### PR TITLE
Mateusz Barański 209269

### PR DESCRIPTION
**Pull request by Matix97**

### CODE REVIEW

File: src/main/java/edu/kis/vh/nursery/FIFORhymer.java [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
-		
-		int ret = temp.countOut();
-		
-		while (!temp.callCheck())
-			
-		countIn(temp.countOut());
-		
-		return ret;
-	}
+    public defaultCountingOutRhymer temp = new defaultCountingOutRhymer();
```
Comment (Lola9614) [inline src/main/java/edu/kis/vh/nursery/FIFORhymer.java]: test

**Review by Lola9614 — COMMENTED** [review]
Koniec testu

File: src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int STACK_CAPACITY = 12;
+    private static final int EMPTY = -1;
+    private static final int MAX_SIZE = 11;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java]: This constant is related to STACK_CAPACITY. Changing only one leads to an error (inconsistency).

File: src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int STACK_CAPACITY = 12;
+    private static final int EMPTY = -1;
+    private static final int MAX_SIZE = 11;
+    private int[] NUMBERS = new int[STACK_CAPACITY];
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java]: Array is not a constant (lowercase)

File: src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    private static final int STACK_CAPACITY = 12;
+    private static final int EMPTY = -1;
+    private static final int MAX_SIZE = 11;
+    private int[] NUMBERS = new int[STACK_CAPACITY];
+
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = -1;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java]: Another constant.

File: src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java [hunk 5]
```diff
@@ -26,15 +26,20 @@ public boolean isFull() {
     public int top() {
         if (isEmpty())
             return -1;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java:28]: Still many constants are not extracted

File: src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java [hunk 6]
```diff
@@ -2,16 +2,16 @@
 
 public class IntLinkedList {
 
-    Node last;
-    int i;
+    private Node last;
+    private int i;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java:6]: Unused private field could be removed.

**Review by kowallus — COMMENTED** [review]
4.5 pts

Specific issues inlined in the code.

**Pull request closed without merging**